web/styles/portico/billing.css

```diff
@@ -1,3 +1,7 @@
+:root {
+    --color-background-modal: hsl(0deg 0% 98%);
+}
```

Comment:
I'm not nearly comfortable with CSS to know whether this is the right way to make this happen, so feedback welcome